plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/storage/KVMStorageProcessor.java

```diff
@@ -1325,6 +1328,8 @@ public Answer deleteSnapshot(final DeleteCommand cmd) {
                     image.snapRemove(snapshotName);
                     s_logger.info("Snapshot " + snap_full_name + " successfully removed from " +
                             primaryPool.getType().toString() + "  pool.");
+                } catch (RbdException e) {
+                    s_logger.error(e.toString() + " - " + ErrorCode.getErrorMessage(e.getReturnValue()));
```

Comment:
Should we maybe include a bit more? Like:

<pre>Failed to remove snapshot X, due to error: Y</pre>

Now we just dump a Exception into the logs which might not always tell a user where it came from.